setup.py

```diff
@@ -54,7 +54,7 @@ def _get_test_env():
 
     def run(self):
 
-        cmd = ['py.test']
+        cmd = ['py.test -vv']
```

Comment:
Is there ever a reason why more verbosity would be bad?
If a test ever fails it's likely one of the first things we're going to do is working out how the ```ganga-ci``` differs from some systems we're developing on. This likely requires some more feedback from assert errors.